packages/server/src/api/bannerRouter.ts

```diff
@@ -54,6 +55,44 @@ interface PuzzlesDataResponse {
     debug?: Debug;
 }
 
+const addLocalLanguageContent = (
+    test: BannerTest,
+    variant: BannerVariant,
+    country: string,
+    moduleName: string,
+): { bannerContent?: BannerContent; mobileBannerContent?: BannerContent } => {
+    const { bannerContent, mobileBannerContent } = variant;
```

Comment:
Ah, should of spotted this much better/simpler routing through approach, this automatcially passes through both  banner contents removing my explicit null checks further down in the props. This stops the  empty content banners occurring when you create a new one (which I now can see I introduced),. Apologies and thankyou for the refactor.